pkgs/development/python-modules/video2midi/default.nix

```diff
@@ -1,24 +1,24 @@
-{ stdenv, fetchFromGitHub, python2Packages, opencv3 }:
+{ stdenv, fetchFromGitHub, pythonPackages, opencv3 }:
 
 let
-  opencv3_ = python2Packages.toPythonModule (opencv3.override {
+  opencv3_ = pythonPackages.toPythonModule (opencv3.override {
     enablePython = true;
-    pythonPackages = python2Packages;
+    pythonPackages = pythonPackages;
     enableFfmpeg = true;
   });
 in stdenv.mkDerivation rec {
```

Comment:
This shouldn't use `stdenv.mkDerivation` but `buildPythonApplication`.

see https://github.com/NixOS/nixpkgs/pull/64993#issuecomment-512955928.